lib/measured/units/dimension.rb

```diff
@@ -0,0 +1,26 @@
+class Measured::Dimension < Measured::Measurable
+
+  conversion.set_base :m,
+    aliases: [:meter, :metre, :meters, :metres]
+
+  conversion.add :cm,
+    aliases: [:centimeter, :centimetre, :centimeters, :centimetres],
+    value: "0.01 m"
```

Comment:
Since this is parsed as `BigDecimal("0.01")` it is just as precise. Nothing is this lib is ever treated as a floating point number.